content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+=========================
+Engineering change orders
+=========================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |BOMs| replace:: :abbr:`BoMs (Bills of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+.. _plm/eco:
+
+Utilize *Engineering Change Orders* (*ECOs*) to track and implement changes to products, and
```

Comment:
_Engineering change orders_ is not a proper noun.
Suggestion around specificity, as well (optional).

```suggestion
Utilize *engineering change orders* (*ECOs*) to track, implement, and revert change versions made to products, and
```